src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -119,6 +120,41 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
  */
 - (void)setPairingDelegate:(id<MTRDevicePairingDelegate>)delegate queue:(dispatch_queue_t)queue;
 
+/**
+ * Set the Delegate for the Device Pairing  as well as the Queue on which the Delegate callbacks will be triggered
+ *
+ * @param[in] delegate The delegate the pairing process should use
+ *
+ * @param[in] queue The queue on which the callbacks will be delivered
```

Comment:
These comments don't seem to go with any method?

Reply:
Removed.